marklogic/internal/util.py

```diff
@@ -0,0 +1,5 @@
+from requests import Response
+
+
+def response_has_no_content(response: Response) -> bool:
+    return response.headers.get("Content-Length") == "0"
```

Comment:
Will this cause a problem if the response doesn't have the header?

Reply:
`get()` is the safe way to access a header; it will return `None`. Whereas `headers["Content-Length"]` will throw a `KeyError` if that key does not exist.